prody/dynamics/sampling.py

```diff
@@ -49,11 +49,11 @@ def sampleModes(modes, atoms=None, n_confs=1000, rmsd=1.0):
     :math:`[r_1^k r_2^k ... r_m^k]` are normally distributed random numbers
     generated for conformation :math:`k` using :func:`numpy.random.randn`.
 
-    RMSD of the new conformation from :math:`R_0` can be calculated as
+    RMSD of the new TEST conformation from :math:`R_0` can be calculated as
```

Comment:
I don't see the word test on http://3.15.148.93/manual/reference/dynamics/sampling.html?highlight=samplemodes#prody.dynamics.sampling.sampleModes